### PR TITLE
Environment + License

### PR DESCRIPTION
**Pull request by akoushke**

### CODE REVIEW

**Comment by adamweeks** [comment]
`build(editor): VSCode Configs` should be `build: add editorconfig`

**Comment by akoushke** [comment]
> `build(editor): VSCode Configs` should be `build: add editorconfig`

i'm not sure which pattern we are following here. I could make this more clear:

`build(VsCode): add configuration` I guess? I prefer to stay with angular convention commits

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+12.7.0
```
Comment (lalli-flores) [inline .nvmrc]: We should use the release label
```suggestion
lts/erbium
```

**Comment by lalli-flores** [comment]
Looks good, will approve once the commits are rebased from master

**Comment by adamweeks** [comment]
`docs(LICENSE): MIT` needs a subject `docs(LICENSE): add MIT license`

**Pull request merged**